components/public-api/gitpod/v1/user.proto

```diff
@@ -69,11 +72,17 @@ message UpdateUserRequest {
     optional google.protobuf.Duration inactivity = 1;
     optional bool disabled_disconnected = 2;
   }
+  optional string dotfile_repo = 9;
```

Comment:
I think we should look into repo vs repository and add something about abbreviations to the guidelines, so that we have it consistently 🙏 
https://github.com/gitpod-io/gitpod/blob/c183ec89baae32ffc4c0766ded7923bd12ca0b73/components/public-api/gitpod/v1/envvar.proto#L49

Reply:
Yep. We can use the "API hardening milestone" for this. cc. @akosyakov 

Also, see Google's recommendations on naming conventions: https://cloud.google.com/apis/design/naming_convention?hl=en#name_abbreviation

Reply:
So, maybe we can align and rename ConfigurationService to RepoConfigService :-P